deploy/olm-catalog/cloud-resources/0.7.2/cloud-resources.v0.7.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,284 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - config.openshift.io
+          resources:
+          - infrastructures
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - persistentvolumes
+          - configmaps
+          verbs:
+          - '*'
+        serviceAccountName: cloud-resource-operator
+      deployments:
+      - name: cloud-resource-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: cloud-resource-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: cloud-resource-operator
+            spec:
+              containers:
+              - command:
+                - cloud-resource-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: cloud-resource-operator
+                - name: TAG_KEY_PREFIX
+                  value: integreatly.org/
+                image: quay.io/integreatly/cloud-resource-operator:0.7.2
```

Comment:
this should be `quay.io/integreatly/cloud-resource-operator:v0.7.2`

Reply:
Sorry thats added now